app/controllers/assignments_controller.rb

```diff
@@ -281,6 +281,29 @@ def recreate_grades
                   notice: "#{pluralize(count, 'grade')} created"
   end
 
+  def update_section_toggles
+    toggle_params = params.permit(:assignment_id, :state, :submission_enabled_toggle_id)
+    toggle = SubmissionEnabledToggle.where(id: toggle_params[:submission_enabled_toggle_id]).first
```

Comment:
You can use `.find(toggle_params[:submission_enabled_toggle_id])` here -- much terser.  But that would throw an exception if not found.   So use `.find_by(id: toggle_params[...])` instead: that has the same effect, but returns nil if not found.